node/type.go

```diff
@@ -26,8 +26,24 @@ func (t Type) IsValid() bool {
 	return ok
 }
 
+// ParseType converts string in a type if possible.
+func ParseType(str string) Type {
+	tp, ok := stringToType[str]
+	if !ok {
+		return 0
```

Comment:
Does `0` indicate the default (e.g. full) or an error condition?

Reply:
Error. 1 stands for Full and 2 for light